app/views/govuk_publishing_components/components/_image_card.html.erb

```diff
@@ -21,7 +21,7 @@
           <%= content_tag(shared_helper.get_heading_level, class: heading_class) do %>
             <% if card_helper.href %>
               <%= link_to card_helper.heading_text, card_helper.href,
-                class: "gem-c-image-card__title-link govuk-link #{brand_helper.color_class}",
+                class: "gem-c-image-card__title-link govuk-link govuk-link--no-underline #{brand_helper.color_class}",
```

Comment:
This is a different method to other ways of setting up CSS classes, but it works.  Thought - at what point do we think it's more understandable to have a `%w[]` array and `<<` into that?
For example:
```ruby
  classes = %w[
    gem-c-image-card__title-link 
    govuk-link 
    govuk-link--no-underline
  ]
  classes << brand_helper.color_class
```

Reply:
I think that point is right now! Updated.